### DISCUSSION TITLE
cables-cli add support for disabling updating of screenshot.png

### DISCUSSION THREAD

**Discussion by mickeyvanolst** [cables-gl/cables_docs#382] [Feature Requests] [answered]
It would be cool to have a `-s` argument that can be added to the cables-cli to not include an updated screenshot.png file

**Comment by steam0r** [comment, marked as answer]
hey, feel free to create a pull request against https://github.com/cables-gl/cables-cli :)

we mainly use it in git controlled projects, so reverts are easy and .gitignore is handy, maybe that works for you too

**Reply by mickeyvanolst** [reply]
Sure, just wonder why you have one to not update the index in that case. But yeah, adding it to the gitignore also works in my case
